src/annoymodule.cc

```diff
@@ -205,7 +205,6 @@ py_an_add_item(py_annoy *self, PyObject *args) {
   }
   self->ptr->add_item(item, &w[0]);
 
-  return PyInt_FromLong(0);
```

Comment:
I think this could cause nasty things – you have a function that should return a PyObject* but without a "return" statement it will just read some random memory off the stack and treat as a PyObject*

In Python returning None is the same as no return statement so I think you can do `return Py_None;` here and everything should work